fe/fe-core/src/main/java/org/apache/doris/persist/ModifyTableDefaultDistributionBucketNumOperationLog.java

```diff
@@ -33,13 +34,23 @@ public class ModifyTableDefaultDistributionBucketNumOperationLog implements Writ
     private long dbId;
     @SerializedName(value = "tableId")
     private long tableId;
+    @SerializedName(value = "type")
+    private DistributionInfoType type;
+    @SerializedName(value = "autoBucket")
+    protected boolean autoBucket;
```

Comment:
The default value of the new fields, which read from the old metadata, is null. To maintain compatibility, those binlogs `type == null` should be filtered in `addBinlog`.